posts/innovative-snowflake-features-architecture.md

```diff
@@ -0,0 +1,126 @@
+---
+authors:
+- Pooja Krishnan
+tags:
+- Cloud
+- Snowflake
+date: 2019-08-07T10:33:00.000Z
+title: "Innovative Snowflake Features Part 1: Architecture"
+image:
+
+---
+
+Earlier this year, Ippon published an [Introduction to Snowflake](https://blog.ippon.tech/introduction-to-snowflake/) post which provided a Snowflake Primer. I recently attended the Snowflake Partner Bootcamp in Baltimore, a four day instructor led training program, and was given the opportunity to learn even more. In this first of three posts, I am going to do an in-depth examination of Snowflake's unique architecture style and some of the benefits resulting from it. Before we get started, I've included a small section on the different Snowflake Editions as well as a link to the Documentation page for them.
+
+---
+# Snowflake Editions
+Snowflake offers a variety of editions which offer differing levels of service and features. These include:
+* Standard
+* Premier
+* Enterprise
+* Enterprise for Sensitive data (ESD)
+* Virtual Private Snowflake (VPS)
+
+Look at [Snowflake Editions](https://docs.snowflake.net/manuals/user-guide/intro-editions.html "Snowflake Editions") in the Snowflake Documentation for more information.
+
+---
+# Why Snowflake?
+Snowflake uses a central data repository for persisted data that is accessible to all compute nodes and processes queries in Massively Parallel Processing compute clusters referred to as Virtual Warehouses. This unique architecture allows both for easy data management as well as performance benefits and increased scalability.
+
+In addition, Snowflake is completely ANSI SQL compliant.
+
+---
+# Architecture:
+Snowflake's Architecture consists of the three layers stated below. In each of the following sections, I'll talk a little bit about what each layer contributes to Snowflake.
+
+The Three Layers:
+1. [Database Storage](## 1. Database Storage)
```

Comment:
The links dont seem to show up correctly for me

Reply:
I'm trying to figure those out right now actually.